.ci/docker/gren/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:14.5.0-stretch-slim
```

Comment:
Could we use this image in other places? if so, I'd move it to the library in a follow-up a publish the Docker container in our registry.